js/networking/auth.js

```diff
@@ -0,0 +1,35 @@
+import { postJson, deleteRequest } from './networking';
+import { createJsonDataAttributes } from './json';
+/*
+Class with methods for handling auth against API.
+*/
+
+const BASE_URL = 'https://sandbox-api.justarrived.xyz';
+const USERS_PATH = '/api/v1/users';
+const SESSIONS_PATH = '/api/v1/users/sessions';
+
+export function signUp(email, password, onSuccess, onError) {
+  const requestJson = createJsonDataAttributes({
+    email,
+    password,
+    consent: true,
+    system_language_id: 38,
+    first_name: 'SomeName',
+    last_name: 'SomeLastName' });
+  postJson(BASE_URL + USERS_PATH,
```

Comment:
_BTW_ pro-tip always add `TODO` comments to this kind of code, so you can easily search for all `TODO` occurrences and fix them before going live. Its sooo tedious to go back and check every file for these kinds of things (which in passing might look like real, live code)